### DISCUSSION TITLE
Custom Admin Theme: there is a way to overwrite `admin` javascript resource?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#14278] [Q&A] [answered]
Good morning everyone, 

I have a customized version of bootstrap and I would like to use this version, instead the one included in the `admin` javascript resource. 

There is a way to spin off OrchardCore bootstrap version from this resource or is it possible to overwrite 'admin' resource to create a new ones without the bootstrap script?

Thank you

**Comment by ns8482e** [comment, marked as answer]
Yes register your version of bootstrap with same name as orchard's but SetVersion greater than orchard's

**Reply by PiemP** [reply]
Thank you @ns8482e, 

your solution can work if `bootstrap` and `popper` are load from separated resources. In version1.6.0 of `TheAdmin` theme the `admin` resource already contain `bootstrap` and `popper` script. 

I believe could be useful to separate `bootstrap`, `popper` and the scripts of the `TheAdmin` theme to allow your solution. Do you believe could be useful to do a PR about it?

I can use your solution to define another version of the `admin` resource that doesn't contain `bootstrap` and `popper`.
